airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/uploader/AbstractBigQueryUploader.java

```diff
@@ -74,6 +75,21 @@ public void upload(final AirbyteMessage airbyteMessage) {
     }
   }
 
+  public void upload(final PartialAirbyteMessage airbyteMessage) {
+    try {
+      writer.write(recordFormatter.formatRecord(airbyteMessage));
+    } catch (final IOException | RuntimeException e) {
+      LOGGER.error(airbyteMessage.toString());
```

Comment:
This is a potential leak of a record, but this is mimicing the existing behavior. This should be revisited.